code/game/machinery/harvester.dm

```diff
@@ -84,6 +101,10 @@
 		return
 	var/mob/living/carbon/C = occupant
 	operation_order = reverseList(C.bodyparts)   //Chest and head are first in bodyparts, so we invert it to make them suffer more
+	if(operation_order.len >= 2 && istype(operation_order[operation_order.len - 1], /obj/item/bodypart/head)) // Swap head and torso, so the head is processed last
```

Comment:
There's a lot of assumptions in this code i don't like. Just find the head and chest then insert them at the end of the ordered list.